docs/src/assets/figures/generate-logo.jl

```diff
@@ -4,23 +4,20 @@ function main()
     Drawing(500, 500, "/tmp/colorslogo.png")
     # transparent background
     origin()
-    juliadarkercolors = [Luxor.darker_green, Luxor.darker_purple, Luxor.darker_red]
-    julialightercolors = [Luxor.lighter_green, Luxor.lighter_purple, Luxor.lighter_red]
+    juliacolors = [Luxor.julia_green, Luxor.julia_purple, Luxor.julia_red]
```

Comment:
What about sorting by their lightness (i.e. from dark to light).
```suggestion
    juliacolors = [Luxor.julia_purple, Luxor.julia_red, Luxor.julia_green]
```